js/src/redux/providers/certifications/reducer.js

```diff
@@ -17,17 +17,25 @@
 const initialState = {};
 
 export default (state = initialState, action) => {
-  if (action.type !== 'addCertification') {
-    return state;
+  if (action.type === 'addCertification') {
```

Comment:
switch statement would be good here as well, since we are only checking the type, default is the state return.